src/Raytracer/Display/DisplayModule.cpp

```diff
@@ -26,10 +97,24 @@ namespace Raytracer {
                 _pixels[index].color = sf::Color(color.r, color.g, color.b);
             }
         }
+    }
+
+    void DisplayModule::displayGuy() {
+        for (const auto &text : _allTexts) { // gestion des textes
+            auto it = _allSfTexts.find(text.id);
+            if (it == _allSfTexts.end())
+                setText(text);
+            else
+                updateText(it->second, text);
+        }
+    }
 
+    void DisplayModule::update(const Image &image) {
         // Dessiner et afficher les pixels
         _window.clear();
```

Comment:
move clear just before the _window.display, in order to don't have black screen if logic take a lot of time

Reply:
rien ne s'affiche si je le met avant, je trouve qu'il est bien placé